client/config.go

```diff
@@ -0,0 +1,43 @@
+package client
+
+import (
+	"time"
+
+	"perun.network/go-perun/wallet"
+
+	"github.com/direct-state-transfer/dst-go/identity"
+)
+
+// TimeoutConfig defines the different timeout configurations.
+type TimeoutConfig struct {
+	//Maximum duration the node will wait until the channel is funded by all participants.
+	//If it expires, channel will be closed.
+	Funding time.Duration
+	//Maximum duration a peer or user can take for responding to requests.
+	//If it expires, the peer or user will be considered offline.
+	Response time.Duration
+	//Duration the node will wait to withdraw funds if a channel was registered with a non final state.
+	//If a channel was registered in final state, funds will be withdrawn immediately.
+	OnChainChallenge time.Duration
+	//Maximum duration the node will wait for receiving a confirmation of blockchain transaction.
+	//If it expires, node will consider the transaction as failed.
+	//Use sufficiently larger values when using the node with a real blockchain network.
+	OnChainTxConfirmation time.Duration
+}
+
+// BlockhainConfig defines blockchain specific configuration parameters.
+type BlockchainConfig struct {
+	NodeAddr  identity.URL   //URL of the blockchain node.
+	AdjAddr   wallet.Address //Address of the Adjudicator Contract.
+	AssetAddr wallet.Address //Address of the Asset Holder Contract.
+}
+
+// ClientConfig defines client specific configuration parameters.
+type ClientConfig struct {
+	ClientID string
+	User     identity.UserConfig
+
+	BC         BlockchainConfig
```

Comment:
Maybe just `Blockchain` since you also called the other ones `User` and `Timeout`?

Reply:
Updated.